xml/depl_nodes.xml

```diff
@@ -6479,7 +6482,197 @@ Apply the &barcl; to a Control Node.
+   <para>
+    &productname; supports integration with &ses; (SES), enabling &ceph; block
+    storage as well as image storage services in &cloud;.
+   </para>
+   <bridgehead renderas="sect2">Enabling SES Integration
+   </bridgehead>
+   <para>
+    Start enabling SES integration by uploading a SES configuration file to
+    &crow;. This file is included in the <literal>crowbar-core</literal>
+    package and can be used with the &crow; UI or CLI
+    (<literal>crowbarctl</literal>). The SES configuration file describes
+    various aspects of the &ceph; environment, and keyrings for each user and
+    pool created in the &ceph; environment for &cloud-crow; services.
+   </para>
+   <bridgehead renderas="sect2">SES Configuration
+   </bridgehead>
+   <para>
+    For SES deployments that are version 5.5 and higher, a Salt runner is used
+    to create all the users and pools. It also generates a YAML
+    configuration that is needed to integrate with &cloud;. The integration
+    runner creates separate users for &o_blockstore;, &o_blockstore; backup
+    (not used by &crow; currently) and &o_img;. Both the &o_blockstore; and
+    &o_comp; services have the same user, because &o_blockstore; needs
+    access to create objects that &o_comp; uses.
+   </para>
+   <para>
+    Configure SES with the following steps:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Login as <literal>root</literal> and run the SES 5.5 Salt runner on the
+      Salt admin host.
+     </para>
+     <screen>&prompt.root;salt-run --out=yaml openstack.integrate prefix=mycloud</screen>
+     <para>
+      The prefix parameter allows pools to be created with the specified
+      prefix. By using different prefix parameters, multiple cloud deployments
+      can support different users and pools on the same SES deployment.
+     </para>
+    </step>
+    <step>
+     <para>
+      A YAML file is created with content similar to the following
+      example:
+     </para>
+     <screen>ceph_conf:
+     cluster_network: 10.84.56.0/21
+     fsid: d5d7c7cb-5858-3218-a36f-d028df7b0673
+     mon_host: 10.84.56.8, 10.84.56.9, 10.84.56.7
+     mon_initial_members: ses-osd1, ses-osd2, ses-osd3
+     public_network: 10.84.56.0/21
+cinder:
+     key: ABCDEFGaxefEMxAAW4zp2My/5HjoST2Y87654321==
+     rbd_store_pool: mycloud-cinder
+     rbd_store_user: cinder
+cinder-backup:
+     key: AQBb8hdbrY2bNRAAqJC2ZzR5Q4yrionh7V5PkQ==
+     rbd_store_pool: mycloud-backups
+     rbd_store_user: cinder-backup
+glance:
+     key: AQD9eYRachg1NxAAiT6Hw/xYDA1vwSWLItLpgA==
+     rbd_store_pool: mycloud-glance
+     rbd_store_user: glance
+nova:
+     rbd_store_pool: mycloud-nova
+radosgw_urls:
+     - http://10.84.56.7:80/swift/v1
+     - http://10.84.56.8:80/swift/v1</screen>
+    </step>
+    <step>
+     <para>
+      Upload the generated YAML file to &crow; using the UI or
+      <literal>crowbarctl</literal> CLI.
+     </para>
+    </step>
+    <step>
+     <substeps>
```

Comment:
I don't think we need the a/b sub-points here. This could be just two paragraphs under point #4.